doc/llvmPatchWindows.diff

```diff
@@ -0,0 +1,407 @@
+ clang/include/clang/AST/DeclCXX.h                | 13 ++++++++++
```

Comment:
Where does this patch come from? Is it also available somewhere else where we could just link to? I'm not sure that shipping clang/llvm patches that will inevitably break at some point and need maintenance is a good strategy.

Reply:
It's basically https://reviews.llvm.org/D69322 just being manually rebased on 11.1.0.. but yeah, I agree that it's not the best strategy.. so for now, will just set a link to the review in the Wiki?